_posts/2018-08-26-eager-dcgan/eager-dcgan.Rmd

```diff
@@ -0,0 +1,460 @@
+---
+title: "Generating digits with Keras and TensorFlow eager execution"
+description: |
+  Generative adversarial networks (GANs) are a popular deep learning approach to generating new entities - images, often. We show how to code them using Keras and TensorFlow eager execution.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: http://www.rstudio.com/
+bibliography: bibliography.bib
+date: 08-26-2018
```

Comment:
You might want to create a custom slug for the bibtex citation (currently it's `keydana2018generating`). Here's how you do this: https://radixpub.github.io/radix-r/citations.html#web-articles